src/texture.rs

```diff
@@ -206,14 +206,14 @@ impl Image {
 pub async fn load_image(path: &str) -> Result<Image, FileError> {
     let bytes = load_file(path).await?;
 
-    Ok(Image::from_file_with_format(&bytes, None))
+    Ok(Image::from_file_with_format(&bytes, None).expect("Valid image required"))
```

Comment:
shouldn't it propogate the error as well?

Reply:
Yes it should, but `from_file_with_format` returns an `ImageError` in it's `Result`, while the current function takes `FileError`. I'm not sure what's the way to handle this (what I usually do is create an new `enum` type with both errors, or just use `anyhow`)

Reply:
I believe to justify the breaking change in the API the errors thing should be solved, I do not want to break the API twice - with this PR and with the future PR fixing this panic

Reply:
Fixed in latest commit. Could you review?

Reply:
Looks good to me! Could you remove irrelevant commits from this PR? And I am ready to merge!

Reply:
Removed unrelated change in Cargo.toml

Reply:
now just ui_skins left :)

Reply:
The changes in ui_skins is necessary to pass tests right? Or should that be in another PR?